### PR TITLE
[ODS-2693] Add integration tests for required extension fields

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json [hunk 1]
```diff
@@ -8589,12 +8589,266 @@
...
+														"v3",
+														"ed-fi",
+														"studentSchoolAssociations"
+													]
+												}
+											},
+											"response": []
+										},
+										{
+											"name": "Create Student Parent or Contact Association",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json:8702]: Add a test without extension required properties values to verify those properties are marked as required

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json [hunk 2]
```diff
@@ -8676,6 +8930,634 @@
 									]
 								}
 							]
+						},
+						{
+							"name": "Required field in a New resource in Extension",
+							"item": [
+								{
+									"name": "Create busRoutes",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json:8938]: Add a test without busRouteDirection and operatingCost values to verify those properties are marked as required

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json [hunk 3]
```diff
@@ -8676,6 +8930,634 @@
...
+												"v3",
+												"ed-fi",
+												"schools"
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "Post School",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json:9142]: Add a test without careerPathwayDescriptor value to verify the property is marked as required

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json [hunk 4]
```diff
@@ -8676,6 +8930,634 @@
...
+												"v3",
+												"ed-fi",
+												"staffs"
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "Create Staff",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json:9334]: Add a test without pets value to verify the property is marked as required

**Review by simpat-jesus — CHANGES_REQUESTED** [review]

**Comment by semalaiappan** [comment]
image
Added Required Field Validation

**Review by simpat-jesus — APPROVED** [review]

**Pull request merged**